electron/main.js

```diff
@@ -165,7 +165,7 @@ function showMainWindow() {
     'height': config.DEFAULT_HEIGHT_MAIN,
     'minWidth': config.MIN_WIDTH_MAIN,
     'minHeight': config.MIN_HEIGHT_MAIN,
-    'autoHideMenuBar': false,
+    'autoHideMenuBar': init.restore('hideMenu', false),
```

Comment:
Can it be `init.restore(!'showMenu', false)` or does it have to be `!init.restore('showMenu', true)`?